src/types/value/contentRelationship.ts

```diff
@@ -45,4 +45,5 @@ export interface FilledContentRelationshipField<
 	isBroken?: boolean
 	data?: DataInterface
 	text?: string
+	variant?: string
```

Comment:
⚠️ #issue: From [this Slack message](https://prismic-team.slack.com/archives/C02L3FN3AJK/p1733937871356369?thread_ts=1733933751.211369&cid=C02L3FN3AJK), it seems we are only supporting `variant` on link fields, not content relationship and link to media fields.

Reply:
I wonder if we need to add the `variant` property in `src/types/value/link.ts` as an intersection. Something like this pseudo-code:

```ts
type LinkField = (WebLinkField | LinkToMediaField | ContentRelationshipField) & {
  variant?: string
}
```

That keeps `variant` out of link to media and content relationship fields.

Reply:
first message: i think what we want is to not add this option to the link to media or content relationship concrete fields, but you can still create a _generic_ link without the select and then in page builder select one of the types

second message: i understood from a conversation with @xrutayisire the other day that one might want to use a concrete version of the field and it should already contain all properties, as they're all exported individually